docs/jd-delivery-api-decision-adapter.md

```diff
@@ -0,0 +1,112 @@
+---
+id: js-delivery-api-decision-adapter
+title: Customise your decision adapter & bucketing
+sidebar_label: Customise decsion adapter
+---
+
+Mojito gives users control over the decision/bucketing engine that assigns subjects to tests and recipes. This is especially useful in an ITP 2.3 world devoid of lasting client-set cookies.
+
+The decision adapter lets you:
+
+ - Bucket subjects consistently across devices (e.g. assigning tests by a customer ID)
+ - Work around ITP 2.3 restrictions (e.g. by using a server-set cookie ID)
+ - Implement your own, preferred PRNG or hash function
+
+It gives you unprecedented control over how subjects are assigned.
+
+## Mojito.options.decisionAdapter
+
+When subjects are bucketed and assigned a recipe ([see `Assign subject by sample rate` on the execution flowchart](js-delivery-api-execution-order#split-test-object-execution-flowchart)), the decisionAdapter is called. This can happen up to three times per experiment ([as described below](#expect-to-handle-multiple-decisions-per-test)).
```

Comment:
The grammar here may be a little confusing. On first read I interpreted this as `decisionAdapter` being called _after_ users are bucketed and assigned, rather than it being called _for_ that action.

Perhaps this would suit: 
> To bucket and assign users to a recipe, `decisionAdapter` is called. ...

Reply:
Oh yes, good catch.